www/front_src/src/Resources/Graph/Performance/index.tsx

```diff
@@ -149,6 +150,7 @@ const PerformanceGraph = ({
   const classes = useStyles({
     canAdjustTimePeriod: not(isNil(adjustTimePeriod)),
     graphHeight,
+    isInTooltip,
```

Comment:
tooltip is a bit out of context within this Component, I prefer "displayHeader" or "displayTitle" instead